src/interfaces/store-interfaces/state-interfaces/registration-state-interface.ts

```diff
@@ -1,4 +1,8 @@
+import { BusinessAddressIF } from '@/interfaces/Registration/business-address'
+
 export interface RegistrationStateIF {
-  valid: boolean
+  defineBusinessValid: boolean
+  peopleRolesValid: boolean
   startDate: string
+  businessAddress: BusinessAddressIF | {}
```

Comment:
Conversely, here, we do use `businessAddress` as a whole, in several places, so it makes sense for it to have its own interface.